client/packages/system/src/Immunisation/VaccineCourseView/DescriptionColumn.tsx

```diff
@@ -41,7 +41,7 @@ export const descriptionColumn = <
           disabled={isDisabled}
           defaultValue={column.accessor({ rowData })}
           onBlur={e => {
-            column.setter({ ...rowData, description: e.target.value });
+            column.setter({ ...rowData, label: e.target.value });
```

Comment:
Are you only calling setter on blur to try reduce the renders? It feels a bit weird UX that the save button only enables once you click out of the input, we're not doing that anywhere else I don't think?

Reply:
Good point - I think this is overhang from an earlier version which didn't have a save button and updated on change